packages/sui-studio/bin/sui-studio-test.js

```diff
@@ -36,7 +37,8 @@ const run = async () => {
             'index.js'
           ),
           program.watch && '--watch',
-          program.ci && '--ci'
+          program.ci && '--ci',
+          `-T ${program.timeout}`
```

Comment:
I suggest to make it conditional and avoid default value in order to not suppose that our runtime uses always 2 seconds
```
          program.timeout && `-T ${program.timeout}`
```

Reply:
Agree!